arroyo-sql/src/lib.rs

```diff
@@ -164,6 +175,83 @@ impl ArroyoSchemaProvider {
     pub fn add_defs(&mut self, source: impl Into<String>, defs: impl Into<String>) {
         self.source_defs.insert(source.into(), defs.into());
     }
+
+    fn get_inlined_source(
+        &self,
+        query: &Statement,
+    ) -> Result<(String, Vec<StructField>, Operator)> {
```

Comment:
nit: indentation is off here

Reply:
This is what `cargo fmt` wants, apparently.

Reply:
for some reason it won't fix this, but if you manually fix it it won't revert it either